tests/test_genericdiffreporter.py

```diff
@@ -163,3 +163,6 @@ def assertFileIsEmpty(self, file_path):
         if file_size != 0:
             msg = "File is not empty: {}" % file_path
             self.fail(msg)
+
```

Comment:
Tiny suggestion: I think it would be nice to keep this test with the other reports (llines 27-38).